CodeSniffer/File.php

```diff
@@ -1321,7 +1319,7 @@ public function getWarningCount()
 
 
     /**
-     * Returns the number of successes recorded.
+     * Returns the number of successes recorded. Currently not in use.
```

Comment:
This comment is irrelevant. It's part of the API that can be used by both custom sniffs and custom reports.